4] guessing game 3/main.py

```diff
@@ -0,0 +1,42 @@
+from flask import Flask, render_template, request, redirect, url_for
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return render_template('welcome.html') 
+
+min = 0
+max = 1000
+guess = (max - min) // 2 + min
+
+@app.route('/index', methods=['GET', 'POST'])
+def game():
+
+    global min, max, guess
```

Comment:
globalni promenne se moc pouzivat neuc, to je bad practise a pouzije se to jen kdyz musis neco hodne ohybat a hacknout. To tady neni  potreba